.github/workflows/pr.yml

```diff
@@ -83,3 +83,79 @@ jobs:
+
+      - uses: the-guild-org/shared-config/setup@v1
+        name: set up env
+        with:
+          node-version-file: .node-version
+
+      - name: generate landing page
+        run: yarn workspace @graphql-mesh/serve-runtime generate-landing-page-html
+
+      - name: Build the project
+        working-directory: ./packages/serve-cli
+        run: yarn bundle-binary
+
+      - name: Package for ${{ matrix.os }} ${{ matrix.arch }}
+        if: matrix.arch == 'x86_64' || matrix.os == 'macos-latest-large'
+        working-directory: ./packages/serve-cli
+        run: ./bundle.sh ${{ matrix.os }} ${{ matrix.arch }}
+
+      - name: Setup QEMU
+        if: matrix.arch == 'aarch64' && matrix.os == 'ubuntu-latest'
+        run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+      - name: Package for Ubuntu aarch64
+        if: matrix.arch == 'aarch64' && matrix.os == 'ubuntu-latest'
+        run: |
+          docker run --rm \
+            -v $(pwd):/workspace \
+            -w /workspace/packages/serve-cli \
+            ubuntu:latest \
+            /bin/bash -c "apt-get update && apt-get install -y build-essential && ./bundle.sh ubuntu-latest aarch64"
+
+      - name: Set artifact name and path
+        id: set-artifact
+        run: |
+          ARTIFACT_DIR=$(find packages/serve-cli -type d -name "graphql-mesh-${{ matrix.arch }}-*")
+          ARTIFACT_NAME=$(basename "$ARTIFACT_DIR")
+          echo "::set-output name=artifact_name::$ARTIFACT_NAME"
+          echo "::set-output name=artifact_path::$ARTIFACT_DIR"
+
+      - name: Upload artifact
+        id: upload_artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ steps.set-artifact.outputs.artifact_name }}
+          path: ${{ steps.set-artifact.outputs.artifact_path }}
+
+      - name: Update PR comment with artifact link
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: 'Node Sea Artifacts Status'
+          message: |
+            The build artifacts of `mesh-serve` as a single executable for various platforms:
+            | Platform               | Status         |
+            |------------------------|----------------|
+            | Ubuntu (x86_64)        | [Download](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/artifacts/${{ steps.upload_artifact.outputs.artifact-id }}) |
+            | Ubuntu (arm64)         | [Download](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/artifacts/${{ steps.upload_artifact.outputs.artifact-id }}) |
+            | macOS (x86_64)         | [Download](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/artifacts/${{ steps.upload_artifact.outputs.artifact-id }}) |
+            | macOS (arm64)          | [Download](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/artifacts/${{ steps.upload_artifact.outputs.artifact-id }}) |
```

Comment:
these will have the same link

Reply:
ooh yes, that's why you couldn't find the mac arm64 binary